node/lib/util/commit.js

```diff
@@ -257,6 +307,17 @@ exports.commit = co.wrap(function *(metaRepo, all, metaStatus, message) {
     const subCommits = {};
     let subsChanged = false;
     const commitSubmodule = co.wrap(function *(name) {
```

Comment:
It's closed over enough variables: `subMessages`, `submodules`, `metaRepo`, and `subCommits`, `all`, `signature`, and `subsChanged`, that I think moving it out would not improve readability -- open to other opinions on this though!